osu.Game/Screens/Play/SongProgress.cs

```diff
@@ -119,7 +119,8 @@ private void load(OsuColour colours, OsuConfigManager config, DrawableRuleset dr
 
             if (drawableRuleset != null)
             {
-                AllowSeeking.BindTo(drawableRuleset.HasReplayLoaded);
+                if (player?.Configuration.AllowUserInteraction == true)
+                    ((IBindable<bool>)AllowSeeking).BindTo(drawableRuleset.HasReplayLoaded);
```

Comment:
This reads a little funny, maybe, but it makes some sense to a degree... I think the biggest throw-off here is `HasReplayLoaded` being bindable. This is really just a set in practice but looks like a strange conditional bind. I don't see how to improve this either, so...

Reply:
Can agree; was just leaving it how it was on `master`.